frontend/composables/useChain.ts

```diff
@@ -109,12 +109,12 @@ async function getAuctionWinner(contract: string): Promise<WinningBid | undefine
 
       let winner = { bid: 0, address: "", };
       //try { winner.address = await auctionContract.getWinner(); } catch (err) { console.log("IGNORE ->" ,err) };
-      try { 
-        const winnerResult = await auctionContract.getWinningBid(); 
+      try {
+        const winnerResult = await auctionContract.getWinningBid();
         winner.bid = winnerResult[0];
         winner.address = winnerResult[1];
-      }
-        catch (err) { /* console.log("IGNORE ->" ,err)  */
+      } catch (err) {
+        console.log("^ IGNORING this Error:", err.reason)
```

Comment:
this tells the user that these errors are expected, since we go over each auction and check for winner (but auctions that didn't end yet will revert)